tests/e2e/performance/csi_tests/test_pvc_snapshot_performance.py

```diff
@@ -509,12 +631,56 @@ def test_pvc_snapshot_performance_multiple_files(
         log.info("Deleting the elastic-search instance")
         self.es.cleanup()
 
+        avg_c_time = statistics.mean(all_results)
+        t_dateset = int(data_set / 3)
+
         log.info(f"Full test report for {interface}:")
         log.info(
             f"Test ran {self.tests_numbers} times, " f"All results are {all_results}"
         )
-        log.info(f"The average creation time is : {statistics.mean(all_results)}")
+        log.info(f"The average creation time is : {avg_c_time}")
         log.info(
             f"Number of Files on the volume : {total_files:,}, "
-            f"Total dataset : {int(data_set / 3)} GiB"
+            f"Total dataset : {t_dateset} GiB"
         )
+
+        full_results.add_key("avg_snapshot_creation_time_insecs", avg_c_time)
+        full_results.all_results["total_files"] = total_files
+        full_results.all_results["total_dataset"] = t_dateset
+        full_results.all_results["creation_time"] = all_results
+
+        # Write the test results into the ES server
+        log.info("writing to elastic search")
```

Comment:
Please consider changing the print to: 
log.info("Writing to elastic search server")